pkgs/tools/graphics/wkhtmltopdf-bin/default.nix

```diff
@@ -1,28 +1,81 @@
-{ stdenv, lib, fetchurl, wkhtmltopdf, xar, cpio }:
+{ autoPatchelfHook
+, cpio
+, freetype
+, zlib
+, openssl
+, dpkg
+, fetchurl
+, gcc-unwrapped
+, lib
```

Comment:
The policy these days is to put `lib` first, then use he usage order for the rest.